DFE-Digital/sts-plan-technology-for-your-school
# feat: update contact link on not found page

## Overview

Addresses ticket [#221101](https://dfe-ssp.visualstudio.com/s190-schools-technology-services/_workitems/edit/221101)


As part of the Zendesk integration we need to update all `contact us` links. We have a hard-coded href in the not found page. I have updated the page so that this url now comes from the `contact us` navigation link component on Contentful. Also updated the `service unavailable` page, so that it uses this navigation link instead of an email link.

## Changes

- Href for contact link on not found page now dynamic - comes from Contentful
- Contentful navigation link used in service unavailable page

## How to review the PR

- Navigate to not found page
- Verify that the `contact-us` link now points to the Zendesk contact form

## Checklist
